ArduPlane/quadplane.cpp

```diff
@@ -1977,6 +1977,17 @@ void QuadPlane::motors_output(bool run_rate_controller)
         attitude_control->rate_controller_run();
     }
 
+    if ((options & OPTION_DISARMED_TILT) || (options & OPTION_DELAY_ARMING)) {
```

Comment:
on final review I noticed a time wrap issue. If you flew a quadplane for 49 days then at the end of that time it would suddenly disarm.
I think we need an additional check that the current spool state is SHUT_DOWN

Reply:
a comment explaining why we're doing this would be good too

Reply:
added the comment
Doesn't an unsigned subtraction behave correctly across wraps?